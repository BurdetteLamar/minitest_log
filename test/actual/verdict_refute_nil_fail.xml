<log>
  <summary verdicts='3' failures='2' errors='0'/>
  <section name='Test' method='verdict_refute_nil?' type='fail'>
    <section name='Args=[nil]'>
      <verdict method='verdict_refute_nil?' outcome='failed' id='0' message='Message 0'>
        <actual class='NilClass' value='nil'/>
        <exception class='Minitest::Assertion' message='Expected nil to not be nil.'>
          <backtrace/>
        </exception>
      </verdict>
    </section>
    <section name='Args=[nil]'>
      <verdict method='verdict_refute_nil?' outcome='failed' id='1' message='Message 1'>
        <actual class='NilClass' value='nil'/>
        <exception class='Minitest::Assertion' message='Expected nil to not be nil.'>
          <backtrace/>
        </exception>
      </verdict>
    </section>
  </section>
  <verdict method='verdict_assert_equal?' outcome='passed' id='__error_count__'>
    <expected class='Integer' value='0'/>
    <actual class='Integer' value='0'/>
  </verdict>
</log>
