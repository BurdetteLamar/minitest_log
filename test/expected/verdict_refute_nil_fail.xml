<log>
  <summary_ verdicts='2' failures='2' errors='0'/>
  <section_ name='Test' method='verdict_refute_nil?' type='fail'>
    <section_ name='Args=[nil, &quot;Message 0&quot;]'>
      <verdict_ method='verdict_refute_nil?' outcome='failed' id='0' message='Message 0'>
        <actual_ class='NilClass' value='nil'/>
        <exception_ class='Minitest::Assertion' message='Expected nil to not be nil.'>
          <backtrace_/>
        </exception_>
      </verdict_>
    </section_>
    <section_ name='Args=[nil, &quot;Message 1&quot;]'>
      <verdict_ method='verdict_refute_nil?' outcome='failed' id='1' message='Message 1'>
        <actual_ class='NilClass' value='nil'/>
        <exception_ class='Minitest::Assertion' message='Expected nil to not be nil.'>
          <backtrace_/>
        </exception_>
      </verdict_>
    </section_>
  </section_>
</log>
